From 3eaf2f3cae6642b5e30f1ee5948714e8ca6e2041 Mon Sep 17 00:00:00 2001
From: Eli Schwartz <eschwartz@archlinux.org>
Date: Thu, 26 Mar 2020 13:47:29 -0400
Subject: [PATCH 17/20] kill off whitespace errors

Trailing whitespace makes my .vimrc unhappy, and serves no useful
purpose. Remove them now, so they don't intermingle with followup commit
changes.
---
 scripts/xdg-utils-common.in       |  4 ++--
 tests/include/desktop_environment |  1 -
 tests/testrun                     | 15 +++++++--------
 3 files changed, 9 insertions(+), 11 deletions(-)

diff --git a/scripts/xdg-utils-common.in b/scripts/xdg-utils-common.in
index b4333e2..ed6c337 100644
--- a/scripts/xdg-utils-common.in
+++ b/scripts/xdg-utils-common.in
@@ -329,7 +329,7 @@ detectDE()
            DE=gnome;
            ;;
          LXDE|Lubuntu)
-           DE=lxde; 
+           DE=lxde;
            ;;
          MATE)
            DE=mate;
@@ -342,7 +342,7 @@ detectDE()
 
     if [ x"$DE" = x"" ]; then
       # fallback to uname output for other platforms
-      case "$(uname 2>/dev/null)" in 
+      case "$(uname 2>/dev/null)" in
         CYGWIN*)
           DE=cygwin;
           ;;
diff --git a/tests/include/desktop_environment b/tests/include/desktop_environment
index 210f516..308906b 100755
--- a/tests/include/desktop_environment
+++ b/tests/include/desktop_environment
@@ -53,4 +53,3 @@ if [ `whoami` = 'root' ] ; then
 else
 	echo "user: normal"
 fi
-
diff --git a/tests/testrun b/tests/testrun
index 4a9f748..c169f73 100755
--- a/tests/testrun
+++ b/tests/testrun
@@ -5,7 +5,7 @@ ROOT_TEST_GROUPS="xdg-desktop-icon xdg-desktop-menu xdg-icon-resource xdg-mime x
 USER_TEST_GROUPS="$ROOT_TEST_GROUPS xdg-email xdg-open xdg-screensaver"
 
 usage() {
-cat << _ENDUSAGE_ 
+cat << _ENDUSAGE_
 Usage:
 $0 [-iIcChRSd:p:f:a] [test|testgroup] [...]
 
@@ -14,7 +14,7 @@ Options:
 
      [-I|-i]   Turn off/on interactive tests. Default: on
                (overrides XDG_TEST_NO_INTERACTIVE)
-  
+
      [-C|-c]   Turn off/on deletion of tempfiles. Default: on
                (overrides XDG_TEST_DONT_CLEANUP)
 
@@ -98,7 +98,7 @@ TEST_LOG="$XDG_TEST_DIR/xdg-test.log"
 export USING_TEST_RUNNER="true"
 
 for MY_TEST_FILE in $TEST_FILES; do
-	. "$MY_TEST_FILE" 
+	. "$MY_TEST_FILE"
 done
 
 if [ -z "$XDG_TEST_SELF_LAUNCH"  ] ; then  # not self launched
@@ -133,7 +133,7 @@ if [ -z "$XDG_TEST_SELF_LAUNCH"  ] ; then  # not self launched
 
 	TEST_STATUS_FILE="$XDG_TEST_DIR/tmp/xdgtest_status-$$"
 	USERCLASS="normal user"
-else 
+else
 	TEST_STATUS_FILE="$XDG_TEST_DIR/tmp/xdgtest_status-$XDG_TEST_SELF_LAUNCH"
 	USERCLASS="root"
 fi
@@ -146,7 +146,7 @@ for t in $TEST_LIST; do
 	ATTEMPT=$((ATTEMPT + 1 ))
 
 	echo -n "$t: "
-	
+
 	test_setup
 
 	# Run test - subshell is necessary to keep things contained.
@@ -167,8 +167,8 @@ for t in $TEST_LIST; do
 		5 ) RESULT=UNTESTED
 		    TOTAL=$((TOTAL - 1 )) ## HACK: we don't want to count tests that don't run.
 		    ;;
-		7 ) RESULT=NORESULT ;; 
-		10 ) RESULT=WARN ;; 
+		7 ) RESULT=NORESULT ;;
+		10 ) RESULT=WARN ;;
 		* ) RESULT="UNKNOWN($CODE)";;
 	esac
 	echo "$RESULT"
@@ -223,4 +223,3 @@ else ## Running su'd. save results.
 		exit 0
 	fi
 fi
-
-- 
2.38.1

